### ISSUE TITLE
minikube addons enable ingress

### ISSUE THREAD

**Issue by Lyudmila-Maltsev-8451** [kubernetes/minikube#13127]
### What Happened?

error: Exiting due to MK_ADDON_ENABLE: run callbacks: running callbacks: [waiting for app.kubernetes.io/name=ingress-nginx pods: timed out waiting for the condition]

### Attach the log file

*
* ==> Audit <==
* |---------|-----------------------|----------|---------|---------|-------------------------------|-------------------------------|
| Command |         Args          | Profile  |  User   | Version |          Start Time           |           End Time            |
|---------|-----------------------|----------|---------|---------|-------------------------------|-------------------------------|
| start   | --vm-driver=hyperkit  | minikube | l782423 | v1.24.0 | Wed, 01 Dec 2021 13:53:47 EST | Wed, 01 Dec 2021 13:55:27 EST |
| start   | --vm-driver=hyperkit  | minikube | l782423 | v1.24.0 | Thu, 02 Dec 2021 13:04:07 EST | Thu, 02 Dec 2021 13:04:11 EST |
| start   | --vm-driver=hyperkit  | minikube | l782423 | v1.24.0 | Fri, 03 Dec 2021 09:24:36 EST | Fri, 03 Dec 2021 09:25:36 EST |
| start   | --vm-driver=hyperkit  | minikube | l782423 | v1.24.0 | Mon, 06 Dec 2021 12:18:35 EST | Mon, 06 Dec 2021 12:19:15 EST |
| start   | --vm-driver=hyperkit  | minikube | l782423 | v1.24.0 | Tue, 07 Dec 2021 10:27:23 EST | Tue, 07 Dec 2021 10:28:23 EST |
| service | mongo-express-service | minikube | l782423 | v1.24.0 | Tue, 07 Dec 2021 14:30:27 EST | Tue, 07 Dec 2021 14:30:27 EST |
| start   | --vm-driver=hyperkit  | minikube | l782423 | v1.24.0 | Wed, 08 Dec 2021 09:08:50 EST | Wed, 08 Dec 2021 09:09:49 EST |
| service | mongo-express-service | minikube | l782423 | v1.24.0 | Wed, 08 Dec 2021 09:34:34 EST | Wed, 08 Dec 2021 09:34:34 EST |
|---------|-----------------------|----------|---------|---------|-------------------------------|-------------------------------|

*
* ==> Last Start <==
* Log file created at: 2021/12/08 09:08:50
Running on machine: CINMAC11762
Binary: Built with gc go1.17.2 for darwin/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I1208 09:08:50.739997   20237 out.go:297] Setting OutFile to fd 1 ...
I1208 09:08:50.740444   20237 out.go:349] isatty.IsTerminal(1) = true
I1208 09:08:50.740447   20237 out.go:310] Setting ErrFile to fd 2...
I1208 09:08:50.740452   20237 out.go:349] isatty.IsTerminal(2) = true
I1208 09:08:50.740559   20237 root.go:313] Updating PATH: /Users/l782423/.minikube/bin
W1208 09:08:50.740707   20237 root.go:291] Error reading config file at /Users/l782423/.minikube/config/config.json: open /Users/l782423/.minikube/config/config.json: no such file or directory
I1208 09:08:50.741087   20237 out.go:304] Setting JSON to false
I1208 09:08:50.804196   20237 start.go:112] hostinfo: {"hostname":"CINMAC11762","uptime":3930,"bootTime":1638968600,"procs":502,"os":"darwin","platform":"darwin","platformFamily":"Standalone Workstation","platformVersion":"11.6","kernelVersion":"20.6.0","kernelArch":"x86_64","virtualizationSystem":"","virtualizationRole":"","hostId":"f5f01906-b3c9-559e-a30a-cb90be1a8600"}
W1208 09:08:50.804333   20237 start.go:120] gopshost.Virtualization returned error: not implemented yet
I1208 09:08:50.826796   20237 out.go:176] 😄  minikube v1.24.0 on Darwin 11.6
I1208 09:08:50.827147   20237 notify.go:174] Checking for updates...
I1208 09:08:50.829265   20237 config.go:176] Loaded profile config "minikube": Driver=hyperkit, ContainerRuntime=docker, KubernetesVersion=v1.22.3
I1208 09:08:50.830569   20237 main.go:130] libmachine: Found binary path at /Users/l782423/.minikube/bin/docker-machine-driver-hyperkit
I1208 09:08:50.830638   20237 main.go:130] libmachine: Launching plugin server for driver hyperkit
"logs.txt" 1772L, 204065B

### Operating System

macOS (Default)

### Driver

HyperKit

**Comment by Pictor13** [comment]
Ensure to be disconnected from VPNs, in case you use one.
And try to `rm -fr ~/.minikube` and/or `minikube delete --all --purge`, before starting a new VM.

**Comment by prezha** [comment]
hi @Lyudmila-Maltsev-8451
thank you for reporting this issue and also for your patience!
i don't have a mac with me to test, so can i kindly ask you to try if pr #13439 would resolve your issue and share results?
thank you in advance!

**Comment by prezha** [comment]
hi @Lyudmila-Maltsev-8451,

we haven't heard back from you and as https://github.com/kubernetes/minikube/pull/13439 was merged, so i'll close this issue now

please feel free to open a new issue if you're still facing problems

/close
